packages/esm-bed-management-app/src/bed-administration/form/add-edit-bed.resource.ts

```diff
@@ -0,0 +1,117 @@
+import { useMemo } from 'react';
+import useSWR from 'swr';
+import { type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+
+interface BedForm {
```

Comment:
It looks like none of the exports in `bed-administration.resource.tsx` are used anywhere in the code. Can we safely delete the file to avoid duplication?